scripts/generateSlimMetadataWrapper.js

```diff
@@ -2,7 +2,7 @@
 // Copyright 2017-2019 @polkadot/types authors & contributors
 // This software may be modified and distributed under the terms
 // of the Apache-2.0 license. See the LICENSE file for details.
-
+process.env.JEST_WORKER_ID = '123';
```

Comment:
is this change helpful?

Reply:
yes.. without this, generation of slim meta data fails..